dotCMS/src/main/java/com/dotmarketing/portlets/workflows/business/WorkflowFactoryImpl.java

```diff
@@ -474,7 +475,7 @@ public void deleteWorkflowTask(WorkflowTask task) throws DotDataException {
 			throw new DotDataException(e.toString());

```

Comment:
I suggest to send the whole exception in the DotDataException constructor instead of e.toString(). Some details can be missed from the stack trace if just e.toString() is sent

Reply:
Yeah this is a practice we need to stop doing. Actually we recently worked on a task about seeking for `throws` like this and include the exception object.

Reply:
That was not changed on this task but either way Ozzy already changed it.